ntbs-service/Pages/Shared/_NotificationLayoutWithSideMenu.cshtml

```diff
@@ -17,33 +17,34 @@
 }
```

Comment:
Is this layout used by all of the edit pages?

Is it worth spreading this out and just applying the logic to all forms on a page, that way we will hit things like `Lab results` etc. for free

Reply:
It will also side-step considerations of stylistic choice between $refs or $el

Reply:
Unresolved this comment, as I think this is something that we should look into. If we change it to just serialize all forms:
e.g. `document.QuerySelectorAll('form')` -> serialise

Then we can get covered for changes in all forms and avoid attaching the component to a specific form too. I'm happy to be convinced away from this idea, if you can think of a good reason to not implement it?

Reply:
I am using query selector now to get all the forms